frontend/src/metabase/visualizations/shared/settings/cartesian-chart.ts

```diff
@@ -77,7 +77,7 @@ export function getDefaultDimensions(
 export function getDefaultMetrics(
   rawSeries: RawSeries,
   settings: ComputedVisualizationSettings,
-) {
+): (string | null)[] {
```

Comment:
I started to see this `(string | null)[]` everywhere and I'd lose track of the value. Could/should we extract this to its own type and import it instead?